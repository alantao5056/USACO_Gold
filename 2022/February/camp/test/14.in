645 903667479
