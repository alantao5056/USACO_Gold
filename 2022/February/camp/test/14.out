398.51658248554973552
